### ISSUE TITLE
could not find function "set_global_graph_attrs"

### ISSUE THREAD

**Issue by Granini077** [rich-iannone/DiagrammeR#282]
Hey Guys, 

after updating DiagrammeR to Version 1.0.0 I have a problem to create a process map with the processmapR package, wich is trying to use one of your functions: 

```
>Log2 %>%
   process_map()
Error in set_global_graph_attrs(., attr = "rankdir", value = "LR", attr_type = "graph") : 
  could not find function "set_global_graph_attrs"
```

In Version 0.9.2 I can find this missing function. The newer version doesn't contains it anymore... 
Just to be shure you are informed :)

**Comment by fmannhardt** [comment]
For other people stumbling over this issue. It has been solved in processmapR:
https://github.com/gertjanssenswillen/processmapR/issues/11

**Comment by rich-iannone** [comment]
Thanks for raising the issue @Granini077 . Looks to be solved now though.

closes #282
